scripts/hid_configurator/configurator_cli.py

```diff
@@ -45,7 +45,13 @@ def perform_dfu(dev, args):
         return
     img_ver_dev = info.get_fw_version()
 
-    img_file = DfuImage(args.dfu_image, info, dev.get_board_name(), dev.get_complete_module_name('dfu')[len('{}/'.format('dfu')):])
+    complete_dfu_module_name = dev.get_complete_module_name('dfu')
+
+    if complete_dfu_module_name == 'dfu':
+        complete_dfu_module_name = None
+    else:
+        complete_dfu_module_name = complete_dfu_module_name[len('{}/'.format('dfu')):]
+    img_file = DfuImage(args.dfu_image, info, dev.get_board_name(), complete_dfu_module_name)
```

Comment:
Can we call the variable `bootloader_variant`, please ? It's no longer complete_dfu_module_name here so that could be misleading.